students/hyska_monika/lesson_12_Debugging/combined_validator.py

```diff
@@ -0,0 +1,35 @@
+import validators
+import csv_functions
+import logging_set
+
+
+def combined_validator():
+    logging_set.create_logsfile('validators_logs.log')
+    # ... set email ...
+    email = input('Put email:')
+    while validators.check_email(email) is False:
+        email = input('Put email:')
+    # ... set password ...
+    password = input('Put password:')
+    while validators.password_check(password) is False:
+        password = input('Put password:')
+    # ... set postcode ...
+    post_code = input('Put postcode: ')
+    while validators.check_postal_code(post_code) is False:
```

Comment:
`condition is False` => `not condition`, also is False checks for identity, so an empty list is not False. To get until-loop in Python and not repeat condition, which is dangerous, use `while True: if condition break`